Add new Color Values

chrisdw opened the pull request:
Add the new color values for Pink, Sky, Lime and Black

chrisdw committed:
Add new Color Values

Add the new color values for Pink, Sky, Lime and Black

chrisdw committed:
Upgrade to latest m Newtonsoft.Json

GeorgeHahn commented:
Worked for me, thanks!

chrisdw committed:
Downgrade Newtonsoft

Commit:
Upgrade all projects to Newtonsoft.Json version 6.0.6. Fix color tests for new list

chrisdw committed:
Move to Strings for colors

As colors are moving to being arbitary a fixed list in an enum won't
work, here so change to a string based dictionary.

jamiepenney commented:
Works for me too

chrisdw commented:
Not sure what's up with the webhook test, all the other failures appear to be due to changes in the test board used by the tests.

GeorgeHahn commented:
Does [this](http://blog.trello.com/introducing-unlimited-labels/) affect this PR?

jamiepenney commented:
More than likely, if you can create whatever labels you like then that field will have to become a string rather than an Enum.

chrisdw committed:
Increase Version

chrisdw committed:
get tests green

Repoint the test at a board I control and get the test green
Boards no longer have a description and the pinned property is
deprecated. The starred property replaces it.
Fix some bugs in action data

chrisdw committed:
Checklist API changes

Fix code and interface to cope with the fact that the Checklist API has
changed

chrisdw committed:
Increase Version Number
